simplified-accounts-database/src/main/java/org/nypl/simplified/accounts/database/AccountsDatabase.kt

```diff
@@ -228,32 +228,34 @@ class AccountsDatabase private constructor(
     }
 
     override fun setAccountProvider(accountProvider: AccountProviderType) {
-      this.setDescription(FunctionType { existing ->
+      this.setDescription(
+        FunctionType { existing ->
 
-        val existingProvider = existing.provider()
-        if (existingProvider.id != accountProvider.id) {
-          throw AccountsDatabaseWrongProviderException(
-            "Provider id ${accountProvider.id} does not match the existing id ${existingProvider.id}"
-          )
-        }
-        if (existingProvider.updated.isAfter(accountProvider.updated)) {
-          logger.warn(
-            "attempted to update provider {} with an older definition",
-            existingProvider.id
-          )
-          return@FunctionType existing
-        }
+          val existingProvider = existing.provider()
```

Comment:
Not sure what's going on here. Another level of indentation?

Reply:
Yeah, it wrapped line 231 to 232, which caused everything below that to be indented one additional level.

Reply:
I think that's due to this:

> Method call formatting
> In long argument lists, put a line break after the opening parenthesis.

https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting